<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoSetAddress</class>
 <widget class="QWidget" name="AutoSetAddress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox{border:2px solid black;};</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="autoSet1Btn">
          <property name="minimumSize">
           <size>
            <width>73</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
          </property>
          <property name="text">
           <string>自动分配1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statuslab1">
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="autoSet2Btn">
          <property name="minimumSize">
           <size>
            <width>73</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
          </property>
          <property name="text">
           <string>自动分配2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyProgressBar" name="progressBar_2">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statuslab2">
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="autoSet3Btn">
          <property name="minimumSize">
           <size>
            <width>73</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
          </property>
          <property name="text">
           <string>自动分配3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyProgressBar" name="progressBar_3">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statuslab3">
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="autoSet4Btn">
          <property name="minimumSize">
           <size>
            <width>73</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
          </property>
          <property name="text">
           <string>自动分配4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyProgressBar" name="progressBar_4">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statuslab4">
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyProgressBar</class>
   <extends>QProgressBar</extends>
   <header>myprogressbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
